qutebrowser/commands/runners.py

```diff
@@ -63,6 +63,11 @@ def _url(tabbed_browser):
         msg += "!"
         raise cmdutils.CommandError(msg)
 
+def _strip_levels(url):
+    """Convenience method to get the 2nd level domain of an url."""
+    tld = url.topLevelDomain()
+    host = url.host()[:-len(tld)].split('.')[-1]
```

Comment:
Needs to be able to handle hosts like `localhost`, `127.0.0.1` and other hosts that don't have a tld.

Returning an empty string is "dangerous" if this is to be used in a pattern. Not sure if raising an exception is acceptable here.